### PR TITLE
tech: [GEN-2609] Add Card ghost variant

### PR DESCRIPTION
**Pull request by Youakeem**
## Describe your changes
Subj

## Justify why they are needed
Will be needed in next PR for the product links on the empty checkout page

## Checklist before requesting a review

- [x] I have performed a self-review of my code

### CODE REVIEW

**Comment by Youakeem** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4777?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4777** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4777?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4776** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4776?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4775** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4775?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4774** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4774?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4773** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4773?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4772** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4772?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @Youakeem and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/ui/src/theme/sprinkles.css.ts [hunk 1]
```diff
@@ -25,7 +25,17 @@ const textColors = {
 
 // Only a fraction of the space tokens are used (the most used ones) so we can reduce
 // the number of classes generated by sprinkles.
-const spaceTokensSubset = createSubset(tokens.space, ['none', 'xxs', 'xs', 'sm', 'md', 'lg', 'xl'])
+const spaceTokensSubset = createSubset(tokens.space, [
```
Comment (guilhermespopolin) [inline packages/ui/src/theme/sprinkles.css.ts:28]: The only one missing now is `xxxs` so maybe it doesn't make sense to extract a subset anymore? Should we just include all space options directly?

**Review by guilhermespopolin — APPROVED** [review]

File: packages/ui/src/theme/sprinkles.css.ts (hunk 1 above)
Comment (Youakeem) [reply packages/ui/src/theme/sprinkles.css.ts:28]: I wasn't sure why we extracted them in the first place so I didn't wanna remove it. But if there aren't any special reasons I don't see why not

Comment (Youakeem) [reply packages/ui/src/theme/sprinkles.css.ts:28]: I completely missed the comment right above it :D 🤦🏻‍♂️

Comment (Youakeem) [reply packages/ui/src/theme/sprinkles.css.ts]: Updated to use space tokens directly

**Comment by Youakeem** [comment]
### Merge activity

* **Oct 8, 3:30 AM EDT**: @Youakeem started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4777).

**Pull request merged**